.github/workflows/ci.yml

```diff
@@ -10,7 +10,7 @@ on: [push, pull_request]
 
 jobs:
   lint-shellcheck:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
@audrium Needs rebase against latest `master`.

Reply:
Can you please also remove trailing whitespaces?

![2020-11-18-151459_2560x1440_scrot](https://user-images.githubusercontent.com/517546/99541308-f5d08b00-29b0-11eb-9172-38ba31682237.png)